articles/machine-learning/how-to-machine-learning-interpretability-automl.md

```diff
@@ -169,6 +193,48 @@ with open("myenv.yml","r") as f:
 
 ```
 
+### Write the Scoring Script
+
+Write the script that will be used to predict on your model and creating inference time explanations.
+
+```python
+%%writefile score.py
+import joblib
+import pandas as pd
+from azureml.core.model import Model
+from azureml.train.automl.runtime.automl_explain_utilities import automl_setup_model_explanations
+
+
+def init():
+    global automl_model
+    global scoring_explainer
+
+    # Retrieve the path to the model file using the model name
+    # Assume original model is named original_prediction_model
```

Comment:
Should `original_prediction_model` be `automl_model`?

Reply:
Done. Thanks.